app/controllers/patients_controller.rb

```diff
@@ -110,8 +110,8 @@ def create
     patient = Patient.new(allowed_params)
 
     # Default to copying *required address into monitored address if monitored address is nil
-    if patient.monitored_address_line_1.nil? || patient.monitored_address_state.nil? ||
-       patient.monitored_address_city.nil? || patient.monitored_address_zip.nil?
+    if patient.monitored_address_line_1.nil? && patient.monitored_address_state.nil? &&
+       patient.monitored_address_city.nil? && patient.monitored_address_zip.nil?
```

Comment:
I can't exactly figure out why this change was needed, can you help me out?

Reply:
great question! so before this change, if any of these fields are `nil` it'll set all the monitored address fields as the actual address. this is a problem for the epi-x format because the monitored address zip is not an imported field which means this condition was always true, resulting in the monitored address to always be overwritten. let me know if that doesn't make sense!

Reply:
Yeah that makes sense! This will also have a minor impact on the UI though right? Now if someone enters just a piece of a monitored address, that partial address will be preserved instead of overwriting the whole thing. Which actually seems more correct to me than sneaky overwrites of user entered values, which is what it seems like we would do before. So looks good to me.